tools/releaseBuild/vstsBuild.ps1

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+[cmdletbinding(DefaultParameterSetName = 'Build')]
+param(
+)
+
+DynamicParam {
+    # Add a dynamic parameter '-Name' which specifies the name of the build to run
+
+    # Get the names of the builds.
+    $buildJsonPath = (Join-Path -path $PSScriptRoot -ChildPath 'build.json')
+    $build = Get-Content -Path $buildJsonPath | ConvertFrom-Json
+    $names = @($build.Windows.Name)
+    foreach ($name in $build.Linux.Name) {
+        $names += $name
+    }
+
+    # Create the parameter attributes
+    $ParameterAttr = New-Object "System.Management.Automation.ParameterAttribute"
+    $ValidateSetAttr = New-Object "System.Management.Automation.ValidateSetAttribute" -ArgumentList $names
+    $Attributes = New-Object "System.Collections.ObjectModel.Collection``1[System.Attribute]"
+    $Attributes.Add($ParameterAttr) > $null
+    $Attributes.Add($ValidateSetAttr) > $null
+
+    # Create the parameter
+    $Parameter = New-Object "System.Management.Automation.RuntimeDefinedParameter" -ArgumentList ("Name", [string], $Attributes)
+    $Dict = New-Object "System.Management.Automation.RuntimeDefinedParameterDictionary"
+    $Dict.Add("Name", $Parameter) > $null
+    return $Dict
+}
+
+begin {
+    $Name = $PSBoundParameters['Name']
+}
+
+end {
+
+    $psReleaseBranch = 'master'
```

Comment:
Do you want to create a branch so you can control when you take changes?

Reply:
I am fine with keeping this as master.